### ISSUE TITLE
agenda.every doesn't work with human interval if job has already run 'once'

### ISSUE THREAD

**Issue by davidminor** [agenda/agenda#1447]
Calling agenda.every with a human interval causes `nextRunAt` to be set to the current date. When saving the job a little later, agenda checks to see if the `nextRunAt` is in the past, which it now will be, and if so will not overwrite an existing job's `nextRunAt`. If there is an existing DB entry for the job that was run once and has no `nextRunAt`, then essentially the job will never get scheduled to run.

So basically, conditions for this bug:

- job has already run as 'once'
- call agenda.every
- use human interval
- skipImmediate is false
